### DISCUSSION TITLE
ILogger LogError in developer-local monitoring solution

### DISCUSSION THREAD

**Discussion by christophwille** [open-telemetry/opentelemetry-dotnet#5244] [Q&A]
I made a small sample to show the problem I am trying to solve - basically, how-to get LogError calls to show in a local monitoring solution. The code that logs the error:

https://github.com/christophwille/OTel-de-Wille/blob/fcc5c423726f5917d5b84704152c7434c630d677/LogErrorSample/Program.cs#L22

The code that does the `builder.Logging.AddOpenTelemetry`:

https://github.com/christophwille/OTel-de-Wille/blob/fcc5c423726f5917d5b84704152c7434c630d677/LogErrorSample/WebApplicationBuilderExtensions.cs#L16-L26

As you can see, this is more or less the same as from the sample in this repository.

* AddConsoleExporter works, error is shown in console
* AddAzureMonitorLogExporter works, error is shown in Application Insights

However, AddOtlpExporter is eluding me with either 

* otel-desktop-viewer --browser 16686
* jaeger-all-in-one --collector.zipkin.host-port=:10411

Do you know / recommend a locally installed monitoring tool that allows me to catch the logs via the AddOtlpExporter method? Tracing works like a charm with those two tools though.

**Comment by cijothomas** [comment]
Jaeger is not a tool that supports OTel Logs, so you need to use a different tool. OpenSearch/Elastic/Loki etc. are some of the options.
I do not know what is desktopviewer.

**Reply by christophwille** [reply]
https://github.com/CtrlSpice/otel-desktop-viewer was the tool in question. I really was looking into super-simple "tools" for local OTLP "monitoring".
